templates/3b_Generic_Cluster_Cells.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<p>We should move segmentation_dir and tiff_dir definitions to the end of the notebook, since they're only needed for visualization</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>